recipes/gradle/LICENSE.txt

```diff
@@ -0,0 +1,420 @@
+
```

Comment:
No need to include the licence in the recipe directory as it's already in the tarball

Reply:
I removed the file.